tests/pcap-file-delete-non-alerts-only-flow-timeout-alert/test.yaml

```diff
@@ -0,0 +1,16 @@
+requires:
+  min-version: 9
+
+command: |
```

Comment:
Why do we need a special command ?

If this is only adding `--set pcap-file.delete-when-done=non-alerts` there is the `args` parameter